lib/event_store_client/mapper/encrypted.rb

```diff
@@ -48,7 +49,34 @@ def serialize(event)
       #   which key should be used as an identifier.
       # *Returns*: Specific event class with all data decrypted
 
-      def deserialize(_event); end
+      def deserialize(event)
+        metadata = serializer.deserialize(event.metadata)
```

Comment:
Just question couldn't you deserialize the whole event at once to avoid separate deserialization of meta and data? Just question have no idea if it makes a big difference

Reply:
Well, I probably could. Don't see too much value of it though.